common/data_refinery_common/migrations/0021_update_qn_bucket.py

```diff
@@ -44,7 +44,7 @@ def update_qn_bucket(apps, schema_editor):
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('data_refinery_common', '0019_sample_is_blacklisted'),
+        ('data_refinery_common', '0020_auto_20190607_1505'),
```

Comment:
Wait no, don't change this. This migration has already been run. Change the migration you're adding in this PR to have this migration as a dependency. (Also your dependency should be number 21, not number 20. This migrations is already number 20).